common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/ElasticSearchClient.scala

```diff
@@ -56,10 +56,10 @@ trait ElasticSearchClient {
     val indexExists = client.admin.indices.prepareExists(index)
                         .execute.actionGet.isExists
 
-    if (!indexExists) createIndex(index)
+    if (!indexExists) createImageIndex(index)
```

Comment:
should the `!indexExists` be renamed as well? Should the `deleteIndex` method be renamed?